shared/src/business/entities/EntityConstants.js

```diff
@@ -265,6 +265,15 @@ const EVENT_CODES_REQUIRING_SIGNATURE = COURT_ISSUED_EVENT_CODES.filter(
   d => d.requiresSignature,
 ).map(pickEventCode);
 
+const NOTICE_OF_ATTACHMENTS_IN_NATURE_OF_EVIDENCE = {
```

Comment:
Wondering if this could follow the pattern that is being using with SYSTEM_GENERATED_DOCUMENT_TYPES.standingPretrialOrder for example?

Reply:
Refactor complete.